iOS/SkyService.xcodeproj/project.pbxproj

```diff
@@ -1355,15 +1349,15 @@
 			repositoryURL = "https://github.com/getditto/DittoSwiftPackage";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 4.0.0;
+				minimumVersion = 4.5.0;
 			};
 		};
 		CC8F6073297927BF0088D5CF /* XCRemoteSwiftPackageReference "DittoSwiftTools" */ = {
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/getditto/DittoSwiftTools";
 			requirement = {
-				branch = v4;
-				kind = branch;
+				kind = upToNextMajorVersion;
+				minimumVersion = 4.4.0;
```

Comment:
Looks like `Package.resolved` isn't being commit, so I have this diff when I build the project.

<img width="1508" alt="image" src="https://github.com/getditto/demoapp-skyservice/assets/26117257/31d09070-440f-48c9-87d5-7be2aaf0029a">